locations/spiders/auchan_pl.py

```diff
@@ -0,0 +1,40 @@
+from urllib.parse import urljoin
+
+from locations.categories import Categories, apply_category
+from locations.storefinders.woosmap import WoosmapSpider
+
+# Types stored at https://api.auchan.com/corp/cms/v4/pl/template/store-types?lang=pl (accessible only from website)
+CATEGORY_MAPPING = {
```

Comment:
Should this mapping be:

```
"123": {"brand": "Auchan Whatever", "brand_wikidata": "Q12345", "shop": "supermarket"},
"456": ...
```

And then lower down in the spider, it becomes `item.update(CATEGORY_MAPPING[store_type]`

That way we're distinguishing between Auchan sub-brands.

Reply:
Makes sense, thanks!